tests/test_url.py

```diff
@@ -62,6 +62,17 @@ def test_safe_url_string(self):
     def test_safe_url_string_unsafe_chars(self):
         safeurl = safe_url_string(r"http://localhost:8001/unwise{,},|,\,^,[,],`?|=[]&[]=|")
         self.assertEqual(safeurl, r"http://localhost:8001/unwise%7B,%7D,|,%5C,%5E,[,],%60?|=[]&[]=|")
+        
+    def test_safe_url_string_quote_path(self):
+        safeurl = safe_url_string(u'http://google.com/"hello"', encoding='utf-8', path_encoding='utf-8', quote_path=True)
```

Comment:
I think it's fine to drop ``encoding='utf-8', path_encoding='utf-8',`` here and below, since these are default values in any case.